### PR TITLE
Dev-4278 OKR > Content ID  > Upgrade from eslint-loader to eslint-webpack-plugin

### PR DESCRIPTION
**Pull request by vcanonjr-freedom**
## Ticket references
- [Dev-4278](https://freedom.myjetbrains.com/youtrack/issue/Dev-4278)

## Summary of changes

Feature:
* Replaced eslint-loader by eslint-webpack-plugin and fixed lint errors

Refactor:
* Ran lint-fix script and other manual changes to get rid of lint warning

## Checklist

- [ ] bugfix
- [x] new feature
- [x] breaking change
- [ ] added unit tests
- [ ] changes introduced has been discussed and approved
- [x] requires manual testing
- [ ] documentation has been updated

## Testing

Setup:
* Remove package-lock.json
* Run `npm i`

To Test:
* Run `npm run lint`. There should be no lint errors
* Run `npx quasar dev`, app should start successfully

Regression Tests on Demo pages
1. https://localhost:8080/table-with-tabs
2. https://localhost:8080/table-with-tabs-2
3. https://localhost:8080/table-without-tabs
4. https://localhost:8080/ffilter-demo
5. https://localhost:8080/fhistory-demo

### CODE REVIEW

**Review by paulyves — APPROVED** [review]

**Review by jonelrempis — APPROVED** [review]

File: src/components/Logout.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <div/>
+    <div />
```
Comment (pandafox) [inline src/components/Logout.vue:2]: Let's disable the git autoformatting?... None of these formats are relevant to the actual change

**Review by pandafox — APPROVED** [review]

File: src/components/Logout.vue (hunk 1 above)
Comment (vcanonjr-freedom) [reply src/components/Logout.vue:2]: @pandafox git autoformatting is not enabled. I manually ran `npm run lint-fix` to fix the lint warnings.

**Comment by paulyves** [comment]
✔`npm run lint`
✔ no errors on all demo pages

**Review by johnurbano12 — APPROVED** [review]
✔️ `npm run lint` result
```js
> anytv-quasar-boilerplate@1.5.3 lint /home/user/Documents/Projects/quasar-boilerplate
> eslint --ext .js,.vue src
```

✔️ No error encountered after `npx quasar dev`
[Test Result Video](https://youtu.be/0XW0D84OAXg)

**Pull request merged**